templates/Vagrantfile.erb

```diff
@@ -126,6 +126,7 @@ Vagrant.configure("2") do |c|
 <% config[:customize].each do |key, value| %>
   <% case config[:provider]
      when "libvirt" %>
+    p.default_prefix = "kitchen-<%= File.basename(config[:kitchen_root]) %>-<%= instance.name %>-<%= SecureRandom.uuid %>"
```

Comment:
wondering if it would be better to modify line 114 to include `when "hyperv", "libvirt"` to get the same behaviour. Currently this would be the wrong place to put this especially without wrapping it in `if key == :default_prefix` block as multiple customize parameters for libvirt would then cause this line to get added to Vagrantfile multiple times. Wrapping it with a special key call would also allow for current functionality and make this an optional addition to the machine name.

Reply:
Line 114 use p.vmname but that afaik is not available on libvirt provider thats why default_prefix was used. Now i understand that if a user overrides using default_prefix it could be duplicated.